lib/helpers.js

```diff
@@ -123,7 +146,9 @@ module.exports = {
    */
   log(msg) {
     /* istanbul ignore next */
-    if (!global.isSilent) console.log(msg)
+    if (global.isSilent) return
+    if (global.isStdout) console.error(msg)
```

Comment:
@GianlucaGuarini agreed to avoid `console` api. BTW, I've reallized that it's not so simple.

- `my-tag.tag -> my-tag.js` should be sent to `stderr`
- `riot-cli:      3.0.0 - https://github.com/riot/cli` should be sent to `stdout`

So we might need one more `helpers.info` or something:

- `helpers.log`: send to stdout
- `helpers.info`: send to stderr without exit
- `helpers.err`: send to stderr and exit